components/global/node--default.vue

```diff
@@ -12,7 +12,10 @@
 <script setup lang="ts">
 defineProps<{
   title?: string;
-  type?: string;
+  type: {
```

Comment:
this is not good. we cannot simply use a wrong value. we should instead make the template cope with the value being conditionally available